File: FinalTaskCalculator/View/ViewComplex.cs
using System;
using System.Text;

namespace View
{
    public class ViewComplex
    {
        public String GetValue(String title)
        {
            Console.WriteLine(title);
            while (true)
            {
                string? temp = Console.ReadLine();
                if (temp != null)
                {
                    return temp;
                }
            }
        }
        public void Print(StringBuilder? data, String? title)
        {
            Console.WriteLine(title + " " + data);
        }

    }
}

File: Seminar5/HomeWork/View/ViewComplex.java
package HomeWork.View;

import java.util.Scanner;

public class ViewComplex {
    Scanner in = new Scanner(System.in);

    public String getValue(String title) {
        System.out.printf("%s", title);
        return in.nextLine();
    }
    public void print(StringBuilder data, String title) {
        System.out.printf("%s %s\n", title, data);
    }

}